text/0110-z-pages.md

```diff
@@ -0,0 +1,35 @@
+# zPages: general direction (#110)
+
+Make zPages a standard OpenTelemetry component.
+
+## Motivation
+
+ZPages or self-introspection debug pages are in-process web pages that display collected data from the process they are attached to. They are used to provide in-process diagnostics without the need of any backend to examine traces or metrics. Various implementations of zPages are widely used in many environments. The standard extensible implementation of zPages in OpenTelemetry will benefit everybody.
+
+## Explanation
+
+This OTEP is a request to get a general approval for zPages development as an experimental feature [open-telemetry/opentelemetry-specification#62](https://github.com/open-telemetry/opentelemetry-specification/pull/632). See [opencensus.io/zpages](https://opencensus.io/zpages/) for the overview of zPages.
+
+## Internal details
+
+This is a request for a general direction approval. There are a few principles for the development:
+
+1. ZPages framework MUST NOT be hardcoded into OpenTelemetry SDK.
```

Comment:
It is not clear whether zPages is a "component", "exporter" or "framework".

Reply:
clarified. added a sentence 

> Implementation of zPages includes multiple components - data collection (sampling, filtering, configuration), storage and aggregation, and a framework to expose this data.


and rephrased bullet points